src/crispy/StrongLRUCache.h

```diff
@@ -22,7 +22,11 @@
 #include <stdexcept>
 #include <vector>
 
+#ifdef __x86_64__
 #include <immintrin.h>
+#elif __aarch64__
+#include "sse2neon.h"
```

Comment:
```suggestion
#include <crispy/sse2neon.h>
```
Maybe?

Reply:
That's what I originally tried, and it worked too. Not sure what's better.